model/src/main/java/org/cbioportal/genome_nexus/model/TranscriptConsequence.java

```diff
@@ -70,6 +70,21 @@ public class TranscriptConsequence
 
     private Map<String, Object> dynamicProps;
 
+    private AlphaMissense alphaMissense;
+
+    @Field("alphamissense")
```

Comment:
Do we need `@Field`?

Reply:
Yes. We need it in the  TranscriptConsequence. And other field not useful I have already deleted.